### PR TITLE
Getting start

### PR DESCRIPTION
**Pull request by vjscjp**
Modification of
 Lab 1 - Shipped Set Up first time
 Lab 2 - Getting Started
 Lab 5 - BootStrap using Spark

### CODE REVIEW

File: labs/shipped-bootstrap-spark/1.md [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
-2. From the **API** tab, choose the Spark buildpack.
+1. We need to create project in Shipped using Spark buildpack, refer to Getting Started section of Shipped document: <a href="http://shipped-cisco.com/shipped/api-docs/build/index.html#walkthroughs" target="_blank">Click here</a>
 
-	![](posts/files/shipped-bootstrap-spark/assets/add-spark-api.png)
+Note : We need to make a small change at service creation step, just select "Cisco Spark" Buildpack under API tab.
 
+![](assets/add-spark-api.png)
 
-3. Continue building your application as normal.
+Rest project creation and deployment section will remain same, <a href="https://ciscoshipped.io/shipped/api-docs/build/index.html#5.-build-your-project" target="_blank">more detail.</a>
```
Comment (dorlok) [inline labs/shipped-bootstrap-spark/1.md]: Is this part of the procedure? 
E.g., Continue to create your project as described in build-your-project.

Comment (npateriya) [inline labs/shipped-bootstrap-spark/1.md]: Short answer: No
We have lot of duplication document on basic workflow. We have similar content at 3 places, as these are developed at different times in lifecycle.
- Learning Labs (shipped-quick-start)
- Getting started guide. 
- Workflow. Above document. (This was first document)

I think we want to covey to user, Only this step is different for spark. rest need to follow same steps as 'shipped-quick-start' learning lab.  Please update doc to reflect same.

**Comment by npateriya** [comment]
@vjscjp Please go ahead and merge the PR. 
Also create a release tag v1.4.1

**Comment by vjscjp** [comment]
Done with merge and release 
https://github.com/CiscoCloud/Shipped-Learning-Labs/releases/tag/v1.4.1

**Pull request merged**